normalizer.py

```diff
@@ -2,18 +2,18 @@
 import config
 import sys
 
-def normalizer(data):
-    minHR = float(sys.maxsize)
-    maxHR = float(-1)
+def normalizer(data, keys):
+    minimum = float(sys.maxsize)
+    maximum = float(-1)
 
-    for hr in data:
+    for value in data:
         try:
-            hr = float(hr)
-            if hr != 0:
-                if hr < minHR:
-                    minHR = float(hr)
-                if hr > maxHR:
-                    maxHR = float(hr)
+            value = float(value)
+            # if value != 0: plus indents
+            if value < minimum:
+                minimum = float(value)
```

Comment:
value already equals float(value) from line 11, so you can just say:

minimum = value